src/main/java/Contacts0.java

```diff
@@ -146,4 +144,12 @@ public static void main(String[] args) {
         }
     }
 
+    private static void showWelcomeMessage() {
```

Comment:
Good job on extracting the lines to the method. You may want to say what the method does through documentation